doc/plugins/org.polarsys.capella.transitions.doc/html/11. Iterative Transitions/11.2. Transitions Between Engineering Phases.html

```diff
@@ -111,7 +111,7 @@ <h3 id="Interface_Transition">Interface Transition</h3>
 				</td>
 				<td>
 					<p>Exchange items defined in a phase can be used in a lower phase. According to preference 
-						<b>Transit allocated Exchange Items while transition of Interfaces, Functions or Components</b>, perform a transition of Interface can also trigger a transition of allocated exchange items. If an allocated exchange item is not transitioned, interface will allocate the exchange item from the upper phase.
+						<b>Transit allocated Exchange Items while transition of Interfaces, Functions, Components or State Machines</b>, perform a transition of Interface can also trigger a transition of allocated exchange items. If an allocated exchange item is not transitioned, interface will allocate the exchange item from the upper phase.
```

Comment:
Can you add a "Note" in the upper State Machine transition section likewise ?

Reply:
Yes, that's a good idea.